Undefined symbols for architecture armv7:   "_UIApplicationWillEnterForegroundNotification"

<!-- DO NOT DELETE
validate_template=true
template_path=.github/ISSUE_TEMPLATE/bug_report.md
-->
### [REQUIRED] Step 1: Describe your environment

  * Xcode version: 11.3.1
  * Firebase SDK version: 6.19.0
  * Firebase Component: Firestore
  * Component version: 1.11.1
  * Installation method: `CocoaPods` (`cocoapods-rome` plugin)

### [REQUIRED] Step 2: Describe the problem

When building the framework I get the following error:

```
Ld /Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Pods.build/Release-iphoneos/FirebaseFirestore.build/Objects-normal/armv7/Binary/FirebaseFirestore normal armv7 (in target 'FirebaseFirestore' from project 'Pods')
    cd /Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/Pods
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -target armv7-apple-ios8.0 -dynamiclib -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.2.sdk -L/Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Release-iphoneos/FirebaseFirestore -F/Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Release-iphoneos/FirebaseFirestore -F/Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Release-iphoneos/BoringSSL-GRPC -F/Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Release-iphoneos/FirebaseCore -F/Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Release-iphoneos/FirebaseCoreDiagnostics -F/Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Release-iphoneos/GoogleDataTransport -F/Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Release-iphoneos/GoogleDataTransportCCTSupport -F/Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Release-iphoneos/GoogleUtilities -F/Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Release-iphoneos/abseil -F/Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Release-iphoneos/gRPC-C++ -F/Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Release-iphoneos/gRPC-Core -F/Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Release-iphoneos/leveldb-library -F/Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Release-iphoneos/nanopb -filelist /Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Pods.build/Release-iphoneos/FirebaseFirestore.build/Objects-normal/armv7/FirebaseFirestore.LinkFileList -install_name @rpath/FirebaseFirestore.framework/FirebaseFirestore -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -dead_strip -Xlinker -object_path_lto -Xlinker /Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Pods.build/Release-iphoneos/FirebaseFirestore.build/Objects-normal/armv7/FirebaseFirestore_lto.o -fembed-bitcode-marker -stdlib=libc++ -fobjc-arc -fobjc-link-runtime -lc++ -framework MobileCoreServices -framework SystemConfiguration -framework absl -framework FirebaseCore -framework Foundation -framework grpcpp -framework leveldb -framework MobileCoreServices -framework nanopb -framework SystemConfiguration -compatibility_version 1 -current_version 1 -Xlinker -dependency_info -Xlinker /Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Pods.build/Release-iphoneos/FirebaseFirestore.build/Objects-normal/armv7/FirebaseFirestore_dependency_info.dat -o /Users/israelsoto/GitHub/SotoiGhost/OldCode/Firebase.CloudFirestore/v1.11.1/build/Pods.build/Release-iphoneos/FirebaseFirestore.build/Objects-normal/armv7/Binary/FirebaseFirestore
Undefined symbols for architecture armv7:
  "_UIApplicationWillEnterForegroundNotification", referenced from:
      firebase::firestore::remote::ConnectivityMonitorApple::ConnectivityMonitorApple(std::__1::shared_ptr<firebase::firestore::util::AsyncQueue> const&) in connectivity_monitor_apple.o
ld: symbol(s) not found for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Last lines of build here: https://gist.github.com/SotoiGhost/513bf83b12a0221d00b4fc4c633c774f

#### Steps to reproduce:

What happened? How can we make the problem occur?
Unzip the file and run `make` (`cocoapods-rome` plugin must be installed to build).

[FirebaseFirestore-v1.11.1.zip](https://github.com/firebase/firebase-ios-sdk/files/4315592/FirebaseFirestore-v1.11.1.zip)

--- a/FirebaseFirestore.podspec
+++ b/FirebaseFirestore.podspec
@@ -74,9 +74,9 @@ Google Cloud Firestore is a NoSQL document database built for automatic scaling,
   s.dependency 'leveldb-library', '~> 1.22'
   s.dependency 'nanopb', '~> 0.3.901'
 
-  s.ios.frameworks = 'MobileCoreServices', 'SystemConfiguration'
+  s.ios.frameworks = 'MobileCoreServices', 'SystemConfiguration', 'UIKit'
   s.osx.frameworks = 'SystemConfiguration'
-  s.tvos.frameworks = 'SystemConfiguration'
+  s.tvos.frameworks = 'SystemConfiguration', 'UIKit'
 
   s.library = 'c++'
   s.pod_target_xcconfig = {

